pom.xml

```diff
@@ -327,11 +326,6 @@
         <artifactId>javax.annotation-api</artifactId>
         <version>1.3.2</version>
       </dependency>
-      <dependency>
-        <groupId>com.fasterxml.jackson.core</groupId>
-        <artifactId>jackson-core</artifactId>
-        <version>${jackson.version}</version>
-      </dependency>
```

Comment:
I'm not 💯 sure I'd actually call this a bug but this is a pretty significant change in behaviour.  We're driving jackson-core versions by whatever jackson-databind indicates, which means we don't have the ability to update jackson-core directly.  That's _probably_ okay, but it is at least worth thinking about.